person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_reader :id, :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Just as suggested by the previous reviewer, kindly make this method private. It is a requirement as you can see [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Private%20method%20of_age%3F%20that%20returns%20true%20if%20%40age%20is%20greater%20or%20equal%20to%2018%20and%20false%20otherwise.)